spec/system/organization_system_spec.rb

```diff
@@ -86,6 +87,13 @@
         expect(page).to have_content(store.name)
       end
 
+      it 'can set single step invite and approve on the organization' do
+        choose('organization[single_step_invite_and_approve]', option: true)
+
+        click_on "Save"
+        expect(page).to have_content("Yes")
```

Comment:
This seems like a really vague test - "Yes" can be anywhere. Can it be more specific?